openfecli/commands/gather.py

```diff
@@ -192,18 +193,18 @@ def _write_ddg(legs, writer, allow_partial):
 
 
 def _write_raw(legs, writer, allow_partial=True):
-    writer.writerow(["leg", "ligand_i", "ligand_j", "DG(i->j) (kcal/mol)",
-                     "MBAR uncertainty (kcal/mol)"])
+    writer.writerow(["leg", "repeat", "ligand_i", "ligand_j",
+                     "DG(i->j) (kcal/mol)", "MBAR uncertainty (kcal/mol)"])
 
     for ligpair, vals in sorted(legs.items()):
         for simtype, repeats in sorted(vals.items()):
-            for m, u in repeats:
+            for rep, (m, u) in enumerate(repeats, 1):
```

Comment:
I think we'd want this to be zero indexed - I'll have a poke around our other outputs later, but at least on the Python layer, we keep everything zero indexed.

Reply:
we will address indexing (whether we want to output it at all) in a follow up PR.